panther_analysis_tool/main.py

```diff
@@ -1303,6 +1333,14 @@ def setup_parser() -> argparse.ArgumentParser:
         "help": "A destination name that may be returned by the destinations function. "
         "Repeat the argument to define more than one name.",
     }
+    # TODO(wey-chiang): Change default to False in v1.0.0
+    use_legacy_mocking_name = "--use-legacy-mocking"
+    use_legacy_mocking_arg: Dict[str, Any] = {
+        "action": "store_true",
+        "default": True,
```

Comment:
I'm not sure I follow the logic here. If someone provides the flag, the value would be `True`, but the default is also `True`, so it would have no effect. I assume that currently someone is unable to activate the new JSON mock support?

Reply:
Doh, brain fart -- good catch